contrib/examples/clusterapi-template.yml

```diff
@@ -141,16 +141,40 @@ objects:
     template:
       metadata:
         labels:
-          # TODO: enforce this in validation
           clusteroperator.openshift.io/machineset: ${CLUSTER_NAME}-master
           clusteroperator.openshift.io/cluster: ${CLUSTER_NAME}
       spec:
         providerConfig:
           value:
             apiVersion: clusteroperator.openshift.io/v1alpha1
+            # Manually populating, this will be set automatically based on our ClusterDeployment.
+            clusterHardware:
+              aws:
+                accountSecret:
+                  name: ${CLUSTER_NAME}-aws-creds
+                keyPairName: ${SSH_KEYPAIR_NAME}
+                region: us-east-1
+                sshSecret:
+                  name: ${CLUSTER_NAME}-ssh-key
+                sshUser: centos
+                sslSecret:
+                  name: ${CLUSTER_NAME}-certs
+            clusterVersionRef: {}
```

Comment:
Remove the fields with empty values. The ones that I see are `clusterVersionRef`, `metadata`, `nodeLabels`, `nodeType`, and `size`.